libmscore/score.h

```diff
@@ -544,7 +545,10 @@ class Score : public QObject, public ScoreElement {
       void addMeasure(MeasureBase*, MeasureBase*);
       void readStaff(XmlReader&);
       bool read(XmlReader&);
-
+      QList<RangeAnnotation*> rangeAnnotations;
```

Comment:
Do we still use this?  neither the member nor the function are accessed anywhere as far as I know